exercises/acronym/acronym_test.cpp

```diff
@@ -0,0 +1,61 @@
+#include "acronym.h"
+#define BOOST_TEST_MAIN
+#include <boost/test/unit_test.hpp>
+
+using namespace std;
+
+BOOST_AUTO_TEST_CASE(basic)
+{
+    const string expected{"PNG"};
+
+    const string actual = acronym::acronym("Portable Network Graphics");
```

Comment:
I think the `expected` and `actual` definitions should be swapped for readability. When I go through and read the tests, it's more logical for me to read it as "The acronym of Portable Network Graphics is expected to be PNG", rather than "I expect to get PNG as the acronym for Portable Network Graphics"

I literally just mean reordering the tests to be:
```
const string actual = ...;

const string expected{...};

BOOST_TEST(expected == actual);
```

Make sense?

Reply:
Sure, will do. I borrowed the skeleton of my tests from the word-count exercise but had noticed that in some other exercises the tests are structured exactly the way you're suggesting.